taipy/core/data/data_node.py

```diff
@@ -505,25 +505,27 @@ def is_valid(self) -> bool:
     def is_up_to_date(self) -> bool:
         """Indicate if this data node is up-to-date.
 
-        Returns:
-            False if a preceding data node has been updated before the selected data node
-            or the selected data is invalid.<br/>
-            True otherwise.
-        """
+    Returns:
+        False if a preceding data node has been updated before the selected data node
+        or the selected data is invalid.<br/>
+        True otherwise.
+    """
```

Comment:
We have a wrong indentation here.
```suggestion
        Returns:
            False if a preceding data node has been updated before the selected data node
            or the selected data is invalid.<br/>
            True otherwise.
        """
```

Reply:
Made the changes👍